test/FunctionalTests/Microsoft.OData.Core.Tests/Build.NetFramework/Microsoft.OData.Core.Tests.csproj

```diff
@@ -454,6 +454,7 @@
     <Compile Include="..\WriterUtilsTests.cs" />
     <Compile Include="..\ScenarioTests\Streaming\ODataJsonLightStreamReadingTests.cs" />
     <Compile Include="..\ScenarioTests\Streaming\ODataJsonLightStreamWritingTests.cs" />
+    <Compile Include="SelectExpandClauseExtensionsTests.cs" />

```

Comment:
can't add this file in "Build.NetFramework". You should add this file in the subfolder of its parent folder.

Reply:
#Resolved